elephant/sta.py

```diff
@@ -314,3 +320,57 @@ def spike_field_coherence(signal, spiketrain, **kwargs):
 
     return (pq.Quantity(sfc, units=pq.dimensionless),
             pq.Quantity(frequencies, units=pq.Hz))
+
+
+def spike_triggered_population_response(spiketrains,
+                                        bin_size=1*pq.ms,
+                                        sigma=5*pq.ms):
+    """
+    Spike triggered population response (stPR) as described in [1]_.
+
+    Parameters
+    ----------
+    spiketrains : list of neo.SpikeTrain
+        list of spike train objects. The spike trains should have the same
+        `t_start` and `t_stop` times.
```

Comment:
I'm a bit slow today.
If the spiketrains should have the same t_start and t_stop, we need to check it and throw the error if necessary.
Take a look at _signals_same_attribute function in asset.
It becomes a neat utility function and I'm thinking to move it to utils.py.
Either copy-paste it or, if you feel adventurous, move this to utils.py and make a public function.